File: CSharp.Functions/Common/Entities/FirmDetails/ProvidersAndIntroducers/ProvidersAndIntroducersBase.cs
namespace Common.Entities
{
    public abstract class ProvidersAndIntroducersBase : CustomerBase
    {
        public long? StartDate { get; set; }
        public string? Status { get; set; }
        public string? DdStatus { get; set; }
        public string? StatusImg { get; set; }
        public string? ProviderImg { get; set; }
    }
}

File: vue.jsonforms/src/entities/providers-and-introducers/ProvidersAndIntroducersBase.ts
import {CustomerBase} from "@/entities/CustomerBase";

export class ProvidersAndIntroducersBase extends CustomerBase {
  public startDate?: number; //this is in epoch
  public status: string = "";
  public ddStatus: string = "";
  public statusImg: string = "";
  public providerImg: string = "";
}